src/components/wrappers/PopupBox.vue

```diff
@@ -15,7 +15,7 @@ export default {
       default: 'bordered',
       validator(value) {
         return value
-          ? ['bordered'].includes(value)
+          ? ['bordered', 'plain'].includes(value)
```

Comment:
oh, nice. it's a great example of thinking about future developing and extending :)